db/schema.rb

```diff
@@ -75,15 +74,14 @@
     t.text "full_time_part_time"
     t.boolean "teaching_scholars"
     t.uuid "dttp_id"
-    t.text "middle_names"
     t.integer "record_type"
+    t.text "middle_names"
     t.text "international_address"
     t.integer "locale_code"
     t.integer "gender"
     t.index ["dttp_id"], name: "index_trainees_on_dttp_id"
     t.index ["gender"], name: "index_trainees_on_gender"
     t.index ["locale_code"], name: "index_trainees_on_locale_code"
-    t.index ["record_type"], name: "index_trainees_on_record_type"
```

Comment:
Why are we removing record_type index? Theres a couple other weird things going on here. I thought the migration was only going to add country to degrees.

Also did you run db:migrate? I was expecting to see the database diagram to be included in this PR